news/index.html

```diff
@@ -83,6 +83,19 @@ <h1><a rel="home" title="Home" href="../index.html" class="floe-header-logo">flo
                 <div class="flc-toc-tocContainer toc"> </div>
                 <h2> News </h2>
 
+                <div class="floe-news-item">
+                    <h3>FLOE Work to Be a Part of the Center on Inclusive Software for Learning</h3>
+                    <p class="floe-date">December 11, 2017</p>
+                    <p>
+                        We're pleased to announce that tools and knowledge from FLOE work on inclusive open education will be a part of to the newly-announced <a href="http://www.cast.org/whats-new/news/2017/center-for-inclusive-software-for-learning.html#.Wi7t9xNSwUE">Center on Inclusive Software for Learning (CISL)</a> as part of the <a href="https://idrc.ocadu.ca/">Inclusive Design Research Centre</a>'s partnership on the project.
+                    </p>
+                        Led by <a href="http://www.cast.org/">CAST</a> and funded by the the U.S. Department of Education's <a href="https://www2.ed.gov/about/offices/list/osers/osep/index.html">Office of Special Education Programs</a>, CISL will support the development of an open-source software suite, industry guidelines and suporting research to ensure K-12 students get engaging personalized learning through Open Education Resources.
+                    </p>
+                    <p>
+                        We look forward to working with CAST and project partner <a href="https://www.sri.com/">SRI International</a> on advancing the state of the art in inclusive open education with FLOE work!
```

Comment:
Same as previous comment about the word "work" at the end of the sentence.

Reply:
The text changes work for me, will do.

The CNAME file removal didn't originate in this PR but I think it should be done at some point - the site is no longer hosted on Github pages, making it surplus. I reverted it for this PR.